cpp/src/slice2swift/Gen.cpp

```diff
@@ -975,7 +967,7 @@ Gen::TypesVisitor::visitEnum(const EnumPtr& p)
     out << sp;
     out << nl << "/// Writes an enumerated value to the stream.";
     out << nl << "///";
-    out << nl << "/// parameter _: `" << name << "` - The enumerator to write.";
+    out << nl << "/// - Parameter v: The enumerator to write.";
```

Comment:
Always document the _parameter_ label, never the _argument_ label.
This is backwards of what you'd expect IMO, but that's how it is.